src/transformers/trainer.py

```diff
@@ -1588,7 +1592,14 @@ def prediction_loop(
             prediction_loss_only if prediction_loss_only is not None else self.args.prediction_loss_only
         )
 
+        if self.args.deepspeed and not self.args.do_train:
+            # In the future we probably can run deepspeed for inference too, but this will require some thinking about how to best run it - since while it works DeepSpeed wasn't designed for inference
```

Comment:
Please stay beneath the 119 char line and make this multi-line.

Reply:
Yes, of course!

But why `make style` hasn't taken care of this?

In genereal the problem is that some parts of our code are auto-formatted, while others aren't. Any reason why these parts aren't auto-formatted too?

Reply:
Done.

Reply:
Black does not touch the comments in general (or strings).

Reply:
Odd, I thought it was. I must have been dreaming it.